### PR TITLE
Post review fixes

### PR DESCRIPTION
**Pull request by valdemon**
As reported at https://github.com/hootsuite/health-checks-api/issues/6#issuecomment-431737952

### CODE REVIEW

File: lib/routes/about.js [hunk 1]
```diff
@@ -17,17 +23,15 @@ module.exports = async (params, service) => ({
         id: service.config.name || service.packageJson.name,
         name: service.config.name || service.packageJson.name,
         description: service.config.description || service.packageJson.description,
-        version: service.packageJson.version || constants.DEFAULT_SERVICE_VERSION,
-        host: service.config.host || getServiceUrl(params),
-        protocol: service.config.protocol || params[constants.HEADER_X_FORWARDED_PROTO] || constants.DEFAULT_PROTOCOL,
+        version: service.config.version || service.packageJson.version || constants.DEFAULT_SERVICE_VERSION,
+        host: service.config.host || os.hostname(),
```
Comment (HootAdam) [inline lib/routes/about.js]: I'm curious about the use case the `service.config.host`. Could it not always be `os.hostname()` or do you use the first for testing?

File: lib/routes/about.js [hunk 2]
```diff
@@ -17,17 +23,15 @@ module.exports = async (params, service) => ({
...
         projectHome: service.config.projectHome || service.packageJson.homepage,
         projectRepo: service.config.projectRepo ||
             (service.packageJson.repository ? service.packageJson.repository.url : constants.DEFAULT_SERVICE_REPO_URL),
-        owners: service.config.owners || [
-            service.packageJson.author && service.packageJson.author.name
-                ? `${service.packageJson.author.name} <${service.packageJson.author.email}>`  
-                : service.packageJson.author
-        ],
+        owners: service.config.owners || authors(service.packageJson),
+        logsLinks: service.config.logsLinks,
```
Comment (HootAdam) [inline lib/routes/about.js]: Nice seeing these in there now :)

File: README.md [hunk 3]
```diff
@@ -106,6 +108,34 @@ The overall health state of the subject service/application is an aggregation of
 
 The module works as a middleware, exposing the [Health Checks API](https://hootsuite.github.io/health-checks-api/) routes via chosen `http server` framework routing system.
 
+### Service details (`about` endpoint)
+
+The [Health Checks API `about` endpoint](https://hootsuite.github.io/health-checks-api/#status-about-get) is supposed to describe the underlying service using this module.
```
Comment (HootAdam) [inline README.md]: Great description

**Comment by valdemon** [comment]
Thanks very much @HootAdam ! New release in on the way.

**Pull request merged**